java/src/main/java/com/google/api/services/samples/youtube/cmdline/live/DeleteLiveChatMessage.java

```diff
@@ -0,0 +1,98 @@
+import java.util.List;
+
+/**
+ * Delets a message from a live broadcast, using OAuth 2.0 to authorize API requests.
+ *
+ * @author Jim Rogers
+ */
+public class DeleteLiveChatMessage {
+
+    /**
+     * Define a global instance of a Youtube object, which will be used
+     * to make YouTube Data API requests.
+     */
+    private static YouTube youtube;
+
+    /**
+     * Deletes a message from a live broadcast.
+     *
+     * @param args The message id to delete (required) followed by the videoId (optional). If the
+     * videoId is given, live chat messages will be retrieved from the chat associated with this
+     * video. If the videoId is not specified, the signed in user's current live broadcast will be
+     * used instead.
+     */
+    public static void main(String[] args) {
+        // Get the message id to delete
+        if (args.length == 0) {
+            System.err.println("No message id specified");
+            System.exit(1);
+        }
+        String messageId = args[0];
+
+        // This OAuth 2.0 access scope allows for read-only access to the
+        // authenticated user's account, but not other types of account access.
+        List<String> scopes = new ArrayList<String>();
+        scopes.add(YouTubeScopes.YOUTUBE_FORCE_SSL);
+        scopes.add(YouTubeScopes.YOUTUBE);
+
+        try {
+            // Authorize the request.
+            Credential credential = Auth.authorize(scopes, "deletelivechatmessage");
+
+            // This object is used to make YouTube Data API requests.
+            youtube = new YouTube.Builder(Auth.HTTP_TRANSPORT, Auth.JSON_FACTORY, credential)
+                .setApplicationName("youtube-cmdline-deletechatmessages-sample").build();
+
+            // Get the liveChatId
+            String liveChatId = GetLiveChatId.getLiveChatId(
+                youtube,
+                args.length == 2 ? args[1] : null);
+            if (liveChatId != null) {
+                System.out.println("Live chat id: " + liveChatId);
+            } else {
+                System.err.println("Unable to find a live chat id");
+                System.exit(1);
+            }
+
+            // Delete the message from live chat
+            YouTube.LiveChatMessages.Delete liveChatDelete =
+                youtube.liveChatMessages().delete(messageId);
+            liveChatDelete.execute();
+            System.out.println("Deleted message id " + messageId);
+        } catch (GoogleJsonResponseException e) {
+            System.err
+                .println("GoogleJsonResponseException code: " + e.getDetails().getCode() + " : "
+                    + e.getDetails().getMessage());
+            e.printStackTrace();
+        } catch (IOException e) {
+            System.err.println("IOException: " + e.getMessage());
+            e.printStackTrace();
+        } catch (Throwable t) {
+            System.err.println("Throwable: " + t.getMessage());
+            t.printStackTrace();
+        }
```

Comment:
I think we could delete the IOException block because it is not handled differently than the Throwable block. I wouldn't simplify any further because I think there is value having these as simple standalone examples.